### ISSUE TITLE
Tb78: dateformat fails if month name / day name variables are included and %language()% set

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplate4#102]
If a language is set explicitly, e.g. `%language(de)%`, the following format tokens make date operations by `%dateformat()%` fail:

```
A - week day
a - shortened week day
B - month name
b - shortened month name
```

**Comment by RealRaven2000** [comment]
Quickfix version:

[smartTemplate-fx-3.3pre3.zip](https://github.com/RealRaven2000/SmartTemplate4/files/5716901/smartTemplate-fx-3.3pre3.zip)

to install, drag into Thunderbird Add-ons Manager.

**Comment by RealRaven2000** [comment]
Fixed in 3.3 released on 03/01/2021
